docs/content/develop/test.md

```diff
@@ -0,0 +1,154 @@
+---
+title: "Add resource tests"
+weight: 40
+aliases:
+  - /docs/how-to/add-mmv1-test
+  - /how-to/add-mmv1-test
+  - /develop/add-mmv1-test
+  - /docs/how-to/add-handwritten-test
+  - /how-to/add-handwritten-test
+  - /develop/add-handwritten-test
+---
+
+# Add resource tests
+
+This page describes how to add tests to a new resource in the `google` or `google-beta` Terraform provider.
+
+For more information about testing, see the [official Terraform documentation](https://developer.hashicorp.com/terraform/plugin/sdkv2/testing/acceptance-tests).
+
+## Before you begin
+
+1. Determine whether your resources is using MMv1 generation or handwritten.
+2. Ensure that your `magic-modules`, `terraform-provider-google`, and `terraform-provider-google-beta` repositories are up to date.
+   ```
+   cd ~/magic-modules
+   git checkout main && git clean -f . && git checkout -- . && git pull
+   cd $GOPATH/src/github.com/hashicorp/terraform-provider-google
+   git checkout main && git clean -f . && git checkout -- . && git pull
+   cd $GOPATH/src/github.com/hashicorp/terraform-provider-google-beta
+   git checkout main && git clean -f . && git checkout -- . && git pull
+   ```
+
+## Add a standard test
```

Comment:
My thought here was that it could be used for adding a single test, and adding multiple tests would be the process of working through the task more than once.

"Standard" was the term I chose for these types of tests, but it can definitely be changed. However, I don't think "create/delete" will work well when we need to use this term in multiple places (because the slash could be ambiguous in some contexts), "basic" already has a different definition, and "non-update" seems a little weird since this is the primary type of test. Is there a problem with "standard"? Maybe "primary" or "coverage"?

Reply:
Coming back to this - I'm ambivalent about plural vs singular. Happy to stick with singular.

My concern with "standard" is that it doesn't clearly mean anything. Like, update tests are also kind of "standard" in that we require them?

I like "coverage" a little better - though update tests are also adding "coverage" for the update process. Maybe "field coverage test" or "create test"? (delete is less relevant since it doesn't matter what fields are set so we could leave that out)

Reply:
I went with your suggestion of "create test" here. I think it describes what is being tested, and makes sense as the counterpart of "update test".